src/Components/Facility/Consultations/DailyRoundsList.tsx

```diff
@@ -51,32 +58,110 @@ export default function DailyRoundsList({ consultation }: Props) {
                   ))}
                 </>
               </PaginatedList.WhenLoading>
-              <PaginatedList.Items<DailyRoundsModel> className="flex grow flex-col gap-3">
-                {(item, items) => {
-                  if (item.rounds_type === "AUTOMATED") {
-                    return (
-                      <VirtualNursingAssistantLogUpdateCard
-                        round={item}
-                        previousRound={items[items.indexOf(item) + 1]}
-                      />
-                    );
-                  }
+              <Timeline
+                className={
+                  "animate-pulse rounded-lg bg-white p-2 py-4 opacity-70 shadow md:px-3"
+                }
```

Comment:
The colors looked muted and had pulse for the entire list

```suggestion
                className="rounded-lg bg-white py-4 shadow"
```